### ISSUE TITLE
Build failure

### ISSUE THREAD

**Issue by glasssaab** [flathub/org.openmw.OpenMW#18]
Noticed build failure - seems to be validating some screenshots linked in an XML file org.openmw.OpenMW.appdata.xml.

```
builddir/files/share/appdata/org.openmw.OpenMW.appdata.xml: FAILED:
• url-not-found : failed to download (HTTP 404: Not Found) [https://old-wiki.openmw.org/images/b/b2/Openmw_0.11.1_launcher_1.png]
• url-not-found : failed to download (HTTP 404: Not Found) [http://old-wiki.openmw.org/images/a/a3/0.40_Screenshot-Balmora_3.png]
• url-not-found : failed to download (HTTP 404: Not Found) [https://old-wiki.openmw.org/images/f/f1/Screenshot_mournhold_plaza_0.35.png]
• url-not-found : failed to download (HTTP 404: Not Found) [https://old-wiki.openmw.org/images/5/5b/Screenshot_Vivec_seen_from_Ebonheart_0.35.png]
```

I think they are located at:

```
https://wiki.openmw.org/images/Openmw_0.11.1_launcher_1.png
https://wiki.openmw.org/images/0.40_Screenshot-Balmora_3.png
https://wiki.openmw.org/images/Screenshot_mournhold_plaza_0.35.png
https://wiki.openmw.org/images/Screenshot_Vivec_seen_from_Ebonheart_0.35.png
```

**Comment by ananace** [comment]
Fixed with e880642

**Comment by ananace** [comment]
Note that GitLab is performing maintenance though, so the next working build will have to be once that's done.
